<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>308</width>
    <height>347</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>308</width>
    <height>347</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="editor.qrc">
    <normaloff>:/newPrefix/icons/config.png</normaloff>:/newPrefix/icons/config.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
	 font: bold gray;
	color: rgb(84, 84, 84);
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 2px
 }

QGroupBox::title {
     font: bold 10px;
     subcontrol-origin: margin;
     subcontrol-position: left top; /* position at the top center */
     padding: 2px;
 }</string>
     </property>
     <property name="title">
      <string>Undo/Redo System</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>System capacity in Megabytes</string>
          </property>
          <property name="text">
           <string>System Capacity (in megabytes)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sizeBox">
          <property name="maximum">
           <number>4000</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="toolTip">
           <string>System occupation</string>
          </property>
          <property name="text">
           <string>Actually in use</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="capacityBar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="erodeoptions">
     <property name="statusTip">
      <string>Erode/Dilate operation options</string>
     </property>
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
	 font: bold gray;
	color: rgb(84, 84, 84);
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 2px
 }

QGroupBox::title {
     font: bold 10px;
     subcontrol-origin: margin;
     subcontrol-position: left top; /* position at the top center */
     padding: 2px;
 }</string>
     </property>
     <property name="title">
      <string>Erode/Dilate/Open/Close Filters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" stretch="1,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>Radius of the structuring element</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string>Kernel Radius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="radiusBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Number of iterations for operation</string>
          </property>
          <property name="statusTip">
           <string>Number of iterations for erode/dilate operation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
	 font: bold gray;
	color: rgb(84, 84, 84);
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 2px
 }

QGroupBox::title {
     font: bold 10px;
     subcontrol-origin: margin;
     subcontrol-position: left top; /* position at the top center */
     padding: 2px;
 }</string>
     </property>
     <property name="title">
      <string>Watershed Filter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="toolTip">
           <string>Flood level</string>
          </property>
          <property name="text">
           <string>Flood Level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="levelBox">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>0.050000000000000</double>
          </property>
          <property name="maximum">
           <double>99.950000000000003</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="visualizationGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> QGroupBox {
	 font: bold gray;
	color: rgb(84, 84, 84);
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 2px
 }

QGroupBox::title {
     font: bold 10px;
     subcontrol-origin: margin;
     subcontrol-position: left top; /* position at the top center */
     padding: 2px;
 }</string>
     </property>
     <property name="title">
      <string>Visualization</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Segmentation opacity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="opacityBox">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>95</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptbutton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acceptbutton</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptbutton</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
